src/main/groovy/com/enonic/defaults/DefaultsPlugin.groovy

```diff
@@ -34,20 +34,19 @@ class DefaultsPlugin
 
     private String resolveRepositoryName()
     {
-        Pattern pattern = Pattern.compile( "git@github.com:enonic/(.*).git" )
-        Matcher matcher = pattern.matcher( this.project.file( ".git/config" ).text )
-        if ( matcher.find() )
-        {
-            return "${matcher.group( 1 )}"
-        }
-        else if ( this.project.rootDir.exists() )
-        {
-            return this.project.rootDir.name
-        }
-        else
+        def gitConfigFile = this.project.file( ".git/config" )
+
+        if ( gitConfigFile.exists() )
         {
-            throw new IllegalStateException( "Not possible to resolve a repository name" )
+            Pattern pattern = Pattern.compile( "git@github.com:enonic/(.*).git" )
+            Matcher matcher = pattern.matcher( gitConfigFile.text )
+            if ( matcher.find() )
+            {
+                return "${matcher.group( 1 )}"
+            }
         }
+
+        return this.project.rootProject.name
```

Comment:
I think CS will fail here as it has customized root project name. Why root project folder doesn't work?